src/components/repo-list-item.component.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+import { ListItem, Icon } from 'react-native-elements';
+
+import { emojifyText, abbreviateNumber } from 'utils';
+import { colors, languageColors, fonts, normalize } from 'config';
+
+type Props = {
```

Comment:
Curious, how do we check for `flow` if we don't have `// @flow` on line 1? I think you already answered this but I don't know exactly where.

Reply:
@alejandronanez we have a lot where there is not this, we need to create a separate PR with the addition of full support Flow throughout the project.

Reply:
Oh...got it! Thanks @lex111